printer_mods/hymness1/Quickdraw_probe_Voron_Switchwire/Klipper_Macros/README.md

```diff
@@ -0,0 +1,79 @@
+    * This doesn't have to be super precise. Z endstop homing is only used for the initial Z homing in order to attach the probe from the dock.
+20. Calculate `last step dock Z coordinate` minus `dock Z coordinate from step 7`.
+21. In `printer.cfg`
+    * Add `[include dock_macros.cfg]`
+    * Enter the correct value for the `position_endstop` in `stepper_z` section by substracting the result in step 20.
+    * Save and restart firmware
+
+# Normal behavior and general notes
+
+The Z endstop microswitch (whether using the modded KeyBak mount or a switch located at Z-max) is only used for initial Z homing if the probe is not already attached to the toolhead. It is a good idea to add `Park_toolhead` to your end_print macro or cancel_print macro, so the probe is virtually always attached to the toolhead.
+
+When homing with the probe attached, terminal will report a wrong value for Z position that should be way higher than supposed, in example `probe at 125.000,84.500 is z=236.425000`. It is normal behavior and is a workaround the fact that you can't have 2 different Z endstops. After homing, position of the toolhead is set to probe z_offset.
+
+Be sure to add `Dock_probe` in your print_start macro after the bed mesh or other probing actions.
+
+Sensorless homing for initial Z homing does work, but I don't recommend it. The KeyBak can be somewhat inconsistent in its tension and can lead to unwanted behavior.
+
+If you don't plan to use a Z endstop microswitch or sensorless homing for initial Z homing, don't change your `endstop_pin` in the `stepper_z` section of `printer.cfg`.
+
+# About the modded KeyBak mount and the endstop microswitch located at Z_min (or sensorless homing at Z_min) and Y travel
+
+The Switchwire has about 230mm of print area in the Y direction, but has around 250mm of Y travel (mine has 257mm), from the endstop to the endblock. About 20 of those millimeters are located outside the bed at the front when the bed is pushed all the way to the rear of the printer. At this location, the nozzle and the part cooling ducts can be clear of the bed when homing at Z_min. The default `printer.cfg` for the VSW with an SKR Mini E3 V2 calls for 0 as position_min and 250 as position_max, whilst it should be, at least that is my case, -25 as position_min and 232 as position_max. When populating the variables in `dock_macros.cfg`, be sure to enter an existing value for `variable_y_min`, and that this value is located outside the bed. If you are using default config for `printer.cfg`, 0 should be a safe coordinate. If you changed the config file for the real travel in the Y axis, -20 should be a safe coordinate. Failure to enter an existing or safe value could at best prevent you from homing with the message `Move out of range`, or at worst crash the nozzle into the bed.
+
+![SW_Y_travel](/printer_mods/hymness1/Quickdraw_probe_Voron_Switchwire/Pictures/SW_Y_travel.png)
```

Comment:
Can you make this a relative reference from this directory?  i.e. it should be

    ![SW_Y_travel](../Pictures/SW_Y_travel.png)

Reply:
Thanks.  Can you also update this README image link?